src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -34,6 +34,8 @@ public abstract class TableValue : ValidFormulaValue
 
         public abstract IEnumerable<DValue<RecordValue>> Rows { get; }
 
+        public abstract bool InMemory { get; }
```

Comment:
>public abstract bool InMemory { get; } [](http://example.com/codeflow?start=8&length=38)

Adding a new property to base class is big - we need to be extremely clear on what exactly these semantics  mean. 
Ie, given an expression with table t, how does it behave differently if t.InMemory=true vs. false.

Reply:
After our discussion I think we should consider capabilities via a set of interfaces to determine what's possible for the rows